modules/change-config-options-after-deployment-central-service.adoc

```diff
@@ -22,9 +22,12 @@ $ helm upgrade -n stackrox \
   -f <path_to_values_private.yaml>
 ----
 <1> If you have modified values that are not included in the `values_public.yaml` and `values_private.yaml` files, include the `--reuse-values` parameter.
-+
-[NOTE]
-====
-You can also specify configuration values using the `--set` or `--set-file` parameters.
+
+When using the `helm upgrade` command, the following guidelines and requirements apply:
+
+* You can also specify configuration values using the `--set` or `--set-file` parameters.
 However, these options are not saved, and it requires you to manually specify all the options again whenever you make changes.
-====
+* Some changes, such as enabling a new component like Scanner V4, require new certificates to be issued for the component. Therefore, you must provide a CA when making these changes.
+** If the CA was generated by the Helm chart during the initial installation, you must retrieve these auto-generated values from the cluster and provide them to the 'helm upgrade' command. A command for retrieving the auto-generated values is included in the post-installation notes of the `central-services` Helm chart.
+** If the CA was generated outside of the Helm chart and provided during the installation of the `central-services` chart, then you must perform that action again when using 'helm upgrade', for example, by using the '--reuse-values' flag with the `helm upgrade` command.
```

Comment:
There shouldn't be plain text following steps in a procedure. Suggest working this into something that tacks onto step 2 (though it's a bit long for that) or moving it up to the intro area.